cluster/HDdistances.py

```diff
@@ -0,0 +1,71 @@
+
+# This file provides functionalities for High dimensionality problems but also for low dimensionality problems
```

Comment:
Instead of a comment, this should preferably be a docstring, making is available to documentation tools like Sphinx.

Reply:
fully agree. i will do it